internal/coreconfig/coreconfig_test.go

```diff
@@ -29,4 +29,6 @@ func TestInitConfigOK(t *testing.T) {
 	Reset()
 
 	assert.Equal(t, 25, config.GetInt(APIDefaultFilterLimit))
+	assert.Equal(t, "localhost", config.GetString(DebugAddress))
```

Comment:
Yeah, this test is for the config scaffolding code, rather than for individual fields - but all good 👍